src/server/client.ts

```diff
@@ -468,6 +468,10 @@ module ts.server {
             throw new Error("Not Implemented Yet.");
         }
 
+        getDocumentHighlights(fileName: string, position: number): DocumentHighlights[] {
+            throw new Error("Not Implemented Yet.");

```

Comment:
Awesome. So does that mean I need to redo the work from yesterday for `getDocumentHighlights`? Should we even expose occurrences on the server end if we're deprecating it?